modules/raveltechBidAdapter.js

```diff
@@ -0,0 +1,72 @@
+// Import the base adapter
+import { spec as baseAdapter } from './appnexusBidAdapter.js'; // eslint-disable-line prebid/validate-imports
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { logInfo } from '../src/utils.js';
+
+const BIDDER_CODE = 'raveltech';
+const URL = 'https://pb1.rvlproxy.net/bid/bid';
+// const URL_SIMPLE = 'https://pb1.rvlproxy.net/bid/simplebid';
+
+export const spec = {
+  code: BIDDER_CODE,
+  gvlid: baseAdapter.GVLID, // use base adapter gvlid
```

Comment:
you need written authorization from @andreiistephens for this

Reply:
We reached our contact at Xandr asking for them to sort this out internally.

Question though: I added `gvlid: baseAdapter.GVLID` because the behaviour of the ravel adapter doesn't change anything to the AppNexus one (except anonymization).

But maybe the correct way would be to not even touch it and remove the line without providing any gvlid?

Reply:
I'm not sure what the anonymize function below does, but wouldn't a publisher want to make sure you had a lawful basis for processing? If you check if xandr does, I'm not sure that's correct?